packages/appeals-service-api/src/repositories/appeals-case-data-repository.js

```diff
@@ -59,6 +59,14 @@ class AppealsCaseDataRepository extends MongoRepository {
 
 		return result;
 	}
+	async postAppealCaseData(caseData) {
+		return await this.upsertManyById([
```

Comment:
Because I don't know the mongo api that well and this is the only method I could find with the word "upsert" in. Does updateOne do upsert in the background?

Reply:
It looks to be a option you can pass in, see updateOne in mongoRepo

Reply:
![image](https://github.com/Planning-Inspectorate/appeal-planning-decision/assets/129742722/a07b7dd8-b6ee-46f2-ace5-9cdeaff53560)

Reply:
Where in the heck is this?

Reply:
Line 83:
https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/main/packages/appeals-service-api/src/repositories/mongo-repository.js

Reply:
Unless I've lost my mind

Reply:
This is what I have at line 83

```
* }
```